spec/features/post_peeps_spec.rb

```diff
@@ -0,0 +1,18 @@
+
+# As a Maker
+# So that I can let people know what I am doing
+# I want to post a message (peep) to chitter
+
+feature "post peeps" do
+  scenario "see peeps" do
+    visit "/"
+    expect(page).to have_content("Chitter:")
+  end
+
+  scenario "add peeps" do
+    visit("/peeps")
```

Comment:
This goes to 'Sinatra doesn’t know this ditty.' What's the difference between a GET and a POST again?